common/script/content/appearance.js

```diff
@@ -0,0 +1,60 @@
+export let defaultAppearancePreferences = {
+  background: {},
+  hair: {
+    bangs: {
+      0: true,
+      1: true,
+      2: true,
+      3: true,
+    },
+    base: {
+      0: true,
+      1: true,
+      3: true,
+    },
+    beard: {
+      0: true,
+    },
+    color: {
+      white: true,
+      brown: true,
+      blond: true,
+      red: true,
+      black: true,
+    },
+    flower: {
+      0: true,
+      1: true,
+      2: true,
+      3: true,
+      4: true,
+      5: true,
+      6: true,
+    },
+    mustache: {
+      0: true,
+    },
+  },
+  shirt: {
+    black: true,
+    blue: true,
+    green: true,
+    pink: true,
+    white: true,
+    yellow: true,
+  },
+  size: {
+    slim: true,
+    broad: true,
+  },
+  skin: {
+    ddc994: true,
+    f5a76e: true,
+    ea8349: true,
+    c06534: true,
+    '98461a': true,
+    915533: true,
+    c3e1dc: true,
+    '6bd049': true,
```

Comment:
As stupid as it is, object attributes that begin with a number and contain letters must be quoted. Those that begin with a letter or properties that are all numbers do not. I can quote them all and disable the eslint rule that objects to unnecessarily quoted items, if you prefer. I was debating about it.